Lib/ldap/ldapobject.py

```diff
@@ -833,6 +832,10 @@ class ReconnectLDAPObject(SimpleLDAPObject):
   * retry_delay: specifies the time in seconds between reconnect attempts.
 
   This class also implements the pickle protocol.
+
+  .. versionadded:: 4.0
+    The exceptions :py:exc:`ldap.UNAVAILABLE`, :py:exc:`ldap.CONNECT_ERROR`,
+    :py:exc:`ldap.TIMEOUT` now also trigger a reconnect.
```

Comment:
Could you also document this is now configurable?

Reply:
Hmm, or how about just updating the docstring to reflect reality and a simple outline of the change here?

Reply:
Also `SERVER_DOWN` is missing from this list now.

Reply:
adjusted.